### ISSUE TITLE
Update wp-config.php

### ISSUE THREAD

**Issue by PatelUtkarsh** [rtCamp/ci-demo#3]
Update wp-config.php for ee4. 

```php
if ( isset( $_SERVER["HTTP_X_FORWARDED_PROTO"] ) && $_SERVER["HTTP_X_FORWARDED_PROTO"] == "https" ) {
    $_SERVER["HTTPS"] = "on";
}
``` 

Props @rahulsprajapati 
https://rtcamp.slack.com/archives/CD47N8XEX/p1544787742012700?thread_ts=1544782624.005700&cid=CD47N8XEX

**Comment by PatelUtkarsh** [comment]
> FYI: this issue will be there in deployer if we are using same for generating wp-config.php
<https://github.com/rtCamp/sys-VIP-Go-Template/issues/1>

<@U0HMZTWUA> you might face this issue for your subdomain site. Please try the solution mentioned in this issue and add it in this new deployer repo.
cc: <@U3BGBANHH> <@U0371Q8K4>
<sub>[View in Slack](https://rtcamp.slack.com/archives/CETRSPVSQ/p1544789182011000)</sub>

**Comment by PatelUtkarsh** [comment]
Closed with #6
